src/saturn_engine/client/worker_manager.py

```diff
@@ -39,9 +39,11 @@ def __init__(
         base_url: str,
         worker_id: Optional[str] = None,
         selector: Optional[str] = None,
+        executors: list[str] | None = None,
     ) -> None:
         self.worker_id: str = worker_id or socket.gethostname()
-        self.selector: Optional[str] = selector
+        self.selector: str | None = selector
```

Comment:
`|` union is Python 3.11 change, you must keep `Optional` for 3.10

Reply:
Lol I just passed 10mins trying to fix that